### PR TITLE
randomized ray length and rotation speed

### PR DESCRIPTION
**Pull request by chang-hs**
# Zero To Mastery - Canvas 2022 🎉🎉

👉 _**Please remove the below and replace with your own values, leaving the headers where they are.**_ 👈

## PR contains:
Simulation of fireworks with randomized starting position, ray number, ray length, and rotation speed

## Breaking changes
None

## Screenshots
icon

### CODE REVIEW

**Comment by chang-hs** [comment]
Hi, I added some more randomization of ray length and rotation speed.
I appreciate your review. Thank you.

File: src/art/chang-hs/script.js [hunk 1]
```diff
@@ -45,32 +45,35 @@ function drawFireWork(){
...
-	ctx.strokeStyle = "green";
-	ctx.arc(x, 250, r, 0, 2*Math.PI, false);
-	ctx.fill();
-	x = x + 5;
-	if (x >= 500 + 100){
-		x = -r;
-	}
-	requestAnimationFrame(draw);
-}
+// function draw(){
```
Comment (sophiabrandt) [inline src/art/chang-hs/script.js:63]: Please don't commit code that you commented out.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

**Comment by chang-hs** [comment]
Thanks for your review. I deleted the commented-out part. Would you review it again?

**Review by auxfuse — APPROVED** [review]
LGTM Thanks for making the changes 🤜

**Pull request merged**